config/opa-service/rego/common.rego

```diff
@@ -7,7 +7,7 @@ import input.attributes.request.http.path as path
 import input.attributes.request.http.headers.authorization as authorization
 
 # Action to method mappings
-scope_method := {"acl:Read": ["GET"], "acl:Write": ["POST"], "acl:Control": ["PUT", "DELETE"]}
+scope_method := {"acl:Read": ["GET"], "acl:Append": ["POST", "PATCH"], "acl:Write": ["POST", "PUT", "DELETE", "PATCH"]}
```

Comment:
just to be sure, can we "overwrite" or extend this outside common.rego if needed?

Reply:
Yes, you can do this by including the `method_matches_action` rule in the other rego files for the individual services and using the mappings you want there. Though this is only for extending, you can't overwrite the above with how rego does things.